concent_api/core/subtask_helpers.py

```diff
@@ -103,15 +111,18 @@ def _update_timed_out_subtask(subtask: Subtask) -> None:
         )
     elif subtask.state == Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name:  # pylint: disable=no-member
         task_to_compute = deserialize_message(subtask.task_to_compute.data.tobytes())
-        # Worker makes a payment from requestor's deposit just like in the forced acceptance use case.
-        bankster.finalize_payment(
-            subtask_id=task_to_compute.subtask_id,
-            concent_use_case=ConcentUseCase.FORCED_ACCEPTANCE,
-            requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
-            provider_ethereum_address=task_to_compute.provider_ethereum_address,
-            subtask_cost=task_to_compute.price,
+        finalize_deposit_claim(
+            subtask_id=subtask.subtask_id,
+            concent_use_case=ConcentUseCase.ADDITIONAL_VERIFICATION,
+            ethereum_address=task_to_compute.requestor_ethereum_address,
```

Comment:
Are these rules consistent with those in Celery tasks?

Reply:
Yes, the same as in `upload_finished` and `verification_result` apart from `MISMATCH` case.